services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -37,16 +37,88 @@ import { fetchPermits } from "@mds/common/redux/actionCreators/permitActionCreat
 import { createDropDownList } from "@mds/common/redux/utils/helpers";
 import { FORM } from "@mds/common/constants/forms";
 import RenderHiddenField from "../forms/RenderHiddenField";
+import AuthorizationSupportDocumentUpload from "./AuthorizationSupportDocumentUpload";
+import { IProjectSummaryDocument } from "@mds/common";
+import {
+  renderTextColumn,
+  renderDateColumn,
+} from "@mds/common/components/common/CoreTableCommonColumns";
+import DocumentTable from "@mds/common/components/documents/DocumentTable";
+import {
+  PROJECT_SUMMARY_DOCUMENT_TYPE,
+  PROJECT_SUMMARY_DOCUMENT_TYPE_CODE,
+  PROJECT_SUMMARY_DOCUMENT_TYPE_CODE_LOCATION,
+} from "../..";
+import { renderCategoryColumn } from "@mds/common/components/common/CoreTableCommonColumns";
+import { MineDocument } from "@mds/common/models/documents/document";
+
+export interface ProjectSummary {
+  project_summary_id: number;
+  mine_guid: string;
+  project_summary_guid: string;
+  status_code: string;
+  submission_date: string;
+  project_summary_description: string;
+  project_guid: string;
+  documents: IProjectSummaryDocument[];
+}
 
-const RenderEMAPermitCommonSections = ({ isAmendment }) => {
-  const purposeLabel = isAmendment
+interface DocumentUploadProps {
```

Comment:
It looks like this isn't being used anywhere.  Is it still needed?

Reply:
Nope, Removed.